resources/user-agents/apps/apple/apple-mail.json

```diff
@@ -373,24 +385,44 @@
         },
         {
           "match": "Mail/* CFNetwork/760*",
+          "properties": {
```

Comment:
I made these match the Mozilla patterns above, which required a couple of changes to some tests.